app/models/preserved_object_primary_moab.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# thin join table to contain a primary CompleteMoab for each PreservedObject
```

Comment:
Humor:   replace comment with "uh, John Martin said we would be needing this ..."

Reply:
serious answer:  this will give us access to created_at and updated_at fields should we need them.

Reply:
haha, @jcoyne and @mjgiarlo provide a good justification in the other thread w/ the easy timestamp field access.  but i also had vague memories of a preservation policies related query where it was a little harder to write a good idiomatic active record query that also resulted in a nice single/atomic SQL query, because there was less ActiveRecord magic available without the model definition.  couldn't remember specifics, but happy to see other benefits i hadn't thought of too.